datus/storage/document/store.py

```
@@ -105,8 +107,19 @@ def __init__(
             on_duplicate_columns="chunk_id",
             schema=schema,
+            db=db,
         )
 
+    def has_data(self) -> bool:
+        """Return True if the document table exists and contains rows."""
+        try:
+            if not self.db.table_exists(self.TABLE_NAME):
+                return False
+            return self._count_rows() > 0
+        except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)